### PR TITLE
Adding Bank name and amount in $ to the msg

### PR DESCRIPTION
**Pull request by eoliveros**

### CODE REVIEW

**Comment by djpnewton** [comment]
lets format the numbers to two decimal places something like like `'{0:0.2f}'.format(invoice.amount_zap/100.0)`

File: templates/utility.html [hunk 1]
```diff
@@ -47,6 +47,7 @@ <h3 class="card-title">{{utility.name}}</h3>
                 <hr/>
                 <form method="post">
                     <input type="hidden" name="zbp_bank_index" value="{{vars.bank_index}}"/>
+                    <input type="hidden" name="zbp_bank_name" value="{{utility.name}}"/>
```
Comment (djpnewton) [inline templates/utility.html]: this should be "zbp_utility_name" no?

File: models.py [hunk 2]
```diff
@@ -97,15 +97,17 @@ class Invoice(db.Model):
     bronze_broker_token = db.Column(db.String(255), nullable=False)
     status = db.Column(db.String(255))
     tx_seen = db.Column(db.Boolean, nullable=False)
+    bank_name = db.Column(db.String(255))
 
-    def __init__(self, email, amount, amount_zap, bronze_broker_token, status):
+    def __init__(self, email, amount, amount_zap, bronze_broker_token, status, bank_name):
```
Comment (djpnewton) [inline models.py]: change to "utility_name"?

File: app.py [hunk 3]
```diff
@@ -370,7 +371,7 @@ def bank_transaction_details(bank_description_item, values):
             details[target] = value
     return details
 
-def invoice_create(utility, details, email, amount):
+def invoice_create(utility, details, email, amount, bank_name):
```
Comment (djpnewton) [inline app.py]: change bank_name to "utility_name"?

File: app.py [hunk 4]
```diff
@@ -173,10 +173,11 @@ def email_invoices_timer_callback():
                     invoice_url = url_for("invoice", token=invoice.token)
                     hostname = urllib.parse.urlparse(invoice_url).hostname
                     sender = "no-reply@" + hostname
+                    formatted_amount = '{0:0.2f}'.format(invoice.amount/100.0)
                     # send email
                     msg = Message('ZAP bill payment status updated', sender=sender, recipients=[invoice.email])
-                    msg.html = 'Invoice <a href="{}">{}</a> has updated to status "{}"'.format(invoice_url, invoice.token, order["status"])
-                    msg.body = 'Invoice {} has updated to status {}'.format(invoice_url, order["status"])
+                    msg.html = 'Invoice <a href="{}">{}</a> has updated the {} invoice with the amount of ${} to status "{}"'.format(invoice_url, invoice.token, invoice.bank_name, formatted_amount, order["status"])
```
Comment (djpnewton) [inline app.py]: change bank_name to "utility_name"?

**Review by djpnewton — CHANGES_REQUESTED** [review]

File: models.py (hunk 2 above)
Comment (eoliveros) [reply models.py]: changed

File: app.py (hunk 3 above)
Comment (eoliveros) [reply app.py]: changed.

File: app.py (hunk 4 above)
Comment (eoliveros) [reply app.py]: changed.

**Comment by eoliveros** [comment]
pushed the commit again... test appears to be good.

will do a rebase if this is ok.

**Pull request merged**